modules/form-field/react/stories/stories_Select.tsx

```diff
@@ -6,7 +6,7 @@ import {controlComponent} from '../../../../utils/storybook';
 
 import FormField from '..';
 import README from '../../../select/react/README.md';
-import {Select, SelectOption} from '@workday/canvas-kit-react-select';
+import {Select, SelectOption} from '../../../select/react/index';
```

Comment:
Open question. We have a bit of a split on the methodology used when relying on other components for a story (dev dependency). 

The majority of places use relative imports like this. The plus side is that it won't cause errors with this script and we don't have to manually manage the dev deps. The downside is that it's less similar to how consumers would use our components.

There are three options. We should probably standardize.

1. Module imports (`import {Something} from '@workday/canvas-kit-react-something'`). Requires adding a dev dependency for each component
2. Relative imports (`import {Something} from '../../../something/react'`). Requires no dev dependency management
3. Universal module (`import {Something} from '@workday/canvas-kit-react'`). Requires only managing the one dev dependency (plus anything that's not exported as part of the universal module).

Thoughts?

Reply:
I believe this change was made so we don't have to build when developing. Otherwise this references the dist folder and a dev would have to build packages when working on a story.

Reply:
@lychyi Maybe originally, but if there is no dist folder (which we never have now), it resolves to the workspace npm link. So that's not really an issue anymore

Reply:
But if you have tested a build for some reason a dist folder exists and it will always default to that right? You could spend a bunch of time trying to fix something without knowing that's actually not the code you're testing. With relative imports there's no indirection involved.

Reply:
Good point. Okay I will standardize on relative imports